cmd/unifi_exporter/main.go

```diff
@@ -9,49 +9,70 @@ import (
 	"net/http"
 	"strings"
 	"time"
+	"io/ioutil"
 
+	"gopkg.in/yaml.v2"
 	"github.com/mdlayher/unifi"
 	"github.com/mdlayher/unifi_exporter"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+type Config struct {
+	TelemetryAddr string `yaml: "telemetryaddr"`
+	MetricsPath string `yaml: "metricspath"`
+	UnifiAddr string `yaml: "unifiaddr"`
+	Username string `yaml: "username"`
+	Password string `yaml: "password"`
+	Site string `yaml: "site"`
+	Insecure bool `yaml: "insecure"`
+	Timeout int `yaml: "timeout"`
+}
```

Comment:
I'd like to keep some logical structure, either by nesting in a yaml object or by prefixing the option names. I.e.

```yaml
listen:
  address:
  port:
unifi:
  address:
  user:
...
```

or

```yaml
listen_address:
listen_port:
unifi_address:
unifi_user:
...
```

People could add options, such as for debugging, logging etc. easier this way and it will be better visible what they're meat for. Also, I'd not drop the defaults, as things such as `listen.address`, `listen.port`, `unifi.insecure` and `unifi.timeout` are unlikely to be modified most of the time.

Reply:
I just looked at why the Travis build failed. Do the struct tags have to be specified like `yaml:"optionname"`, i.e. without the space after the colon?